### DISCUSSION TITLE
Performance issue on `HashProbe::getBuildSideOutput`

### DISCUSSION THREAD

**Discussion by SeaRise** [facebookincubator/velox#10636] [General]
HashProbe build side output handles only single concurrency
https://github.com/facebookincubator/velox/blob/b4ea92c3fc9992db66670ef7c6d52d1df0c03379/velox/exec/HashProbe.cpp#L908-L914

Is it possible for multiple operators to call listRows on different tables for parallel execution?
https://github.com/facebookincubator/velox/blob/b4ea92c3fc9992db66670ef7c6d52d1df0c03379/velox/exec/HashTable.cpp#L1865-L1892

**Comment by kevincmchen** [comment]
@rui-mo  please take a look
